Index.html

```diff
@@ -1,122 +1,124 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Document</title>
-</head>
-
-    <body>
-        <main id="main">
-            <h1 id="title">Dr. Norman Borlaug</h1>
-            <p>The man who saved a billion lives</p>
-            <p>and survived</p>
-            <figure id="img-div">
```

Comment:
Kindly adopt classes as reference to most of your elements . using classes over IDs usually guarantees adoption of the standard HTML semantics. 😉